src/generators/cli.ts

```diff
@@ -1,5 +1,5 @@
 import {execSync} from 'child_process'
-import * as fs from 'fs'
+import * as fs from 'fs-extra'
```

Comment:
Should we not change the `writeJSON` on [line 221](https://github.com/oclif/oclif/pull/1051/files#diff-bf715a9a4f63adf855cca52cc2b37014d190db747585b36a9ad333485ee015f6R221) to have the `{spaces: 2}` option passed in?

Reply:
Or should we leave this import statement alone?

Reply:
no, "this.fs" is not the same fs.  In fact, I'm gonna switch this import back even though it shouldn't matter